README.md

```diff
@@ -1,6 +1,6 @@
 # selectize.js
 
-[![NPM version](http://img.shields.io/npm/v/selectize.svg?style=flat)](https://www.npmjs.org/package/selectize)
+[![CDNJS version](http://img.shields.io/cdnjs/v/selectize.js.svg?style=flat)](https://cdnjs.com/libraries/selectize.js) [![NPM version](http://img.shields.io/npm/v/selectize.svg?style=flat)](https://www.npmjs.org/package/selectize)
```

Comment:
Can there be a line break before the NPM badge? Also, putting it *after* the NPM badge would probably make sense actually.